### DISCUSSION TITLE
Little Capture Loss, could someone confirm my newbie findings?

### DISCUSSION THREAD

**Discussion by GeorgePatches** [Security-Onion-Solutions/securityonion#13437] [Unsupported Versions] [answered]
Hi, I'm playing with SO in my home lab and I seem to be getting a little bit of capture loss.  I thought it was the 20 year old Intel 82571 I was using, but swapping to a much newer Intel I350 or I211-V doesn't seem to had made any difference.  I found this documentation about capture loss (https://docs.securityonion.net/en/2.4/zeek.html#packet-loss-and-capture-loss) and from that is sounds like my switch (Unifi-16-lite) is dropping some of the SPAN traffic.  If someone more experienced could confirm my assessment, I'd appreciate it. 

![Screenshot from 2024-08-05 12-22-13](https://github.com/user-attachments/assets/811f0c68-97d7-4dbc-9c74-2ed21515748d)

**Comment by InfosecGoon** [comment, marked as answer]
That's correct - Capture Loss is Zeek noticing abnormalities in the traffic (SYN without a corresponding FIN to end the conversation, that sort of thing). Loss of under one percent is pretty typical, especially if you're using a SPAN rather than a TAP.

**Reply by GeorgePatches** [reply]
Thanks, the only thought behind me chasing 0.0% is that my internet is ~300Mbit, so my up and down should fit on a single gigabit's transmit.  That lead me down a path of trying to make a SPAN port on my pfSense and that just hasn't worked at all like I was expecting.  Sounds like I should just quit while I'm ahead, losing a little is fine.
